server/sonar-web/src/main/js/app/styles/init/misc.css

```diff
@@ -600,3 +600,8 @@ th.huge-spacer-right {
 .break-word {
   word-break: break-word;
 }
+
+.min-width-fit-content {
+  min-width: fit-content;
```

Comment:
It's better to use `white-space: nowrap;` style in our case. 

Note, SQ already has nowrap style:

```
.nowrap {
  white-space: nowrap !important;
}
```

Reply:
Updated.